scripts/tests/test_render_snyk_report.py

```
@@ -0,0 +1,311 @@
+                        "json_path": str(iac),
+                        "html_path": None,
+                        "source_ref": "k8s",
+                    },
+                ]
+            },
+            indent=2,
+        ),
+        encoding="utf-8",
+    )
+
+    result = subprocess.run(
```

Ruff (0.15.5)

[error] `subprocess` call: check for execution of untrusted input

(S603)